experimental/packages/otlp-transformer/src/metrics/index.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { InstrumentationLibrary } from '@opentelemetry/core';
+import type { Resource } from '@opentelemetry/resources';
+import type { MetricRecord } from '@opentelemetry/sdk-metrics-base';
```

Comment:
Are we going to update the interface to the wip one, or work on it in a follow-up PR?

I'm fine with doing it in a follow-up PR, since the interfaces in the wip package are still going to be changed, like https://github.com/open-telemetry/opentelemetry-js/pull/2829.

Reply:
I would really like to get this moving for the trace exporters. I can remove the metrics stuff from the index.ts export so that the public interface only contains the trace transformer?

Reply:
Sounds good to me! I'd really want to see this huge work getting landed sooner too.